tools/interop_matrix/README.md

```diff
@@ -30,11 +30,17 @@ from specific releases/tag, are used to test version compatiblity between gRPC r
   - `LANG=go KEEP_IMAGE=1 ./create_testcases.sh` will generate `./testcases/go__master` and keep the local docker image so it can be invoked simply via `./testcases/go__master`.  Note: remove local docker images manually afterwards with `docker rmi <image_id>`.
 - Stage and commit the generated test case file `./testcases/<lang>__<release>`.
 
-## Instructions for running test cases against a GCR image
+## Instructions for running test cases against GCR images
+- Run `tools/interop_matrix/run_interop_matrix_tests.py`.  Useful options:
+  - `--release` specifies a git release tag.  Make sure the GCR images with the tag have been created using `create_matrix_images.py` above.
+  - `--language` specifies a language.
+  For examle, To test all languages for all gRPC releases across all runtimes, do `tools/interop_matrix/run_interop_matrix_test.py --release=all`.
```

Comment:
Fixed the typo.  --release=master --language=all are the defaults. Updated the doc to clarify.